test/unit/audio-bandwidth-test.js

```diff
@@ -0,0 +1,351 @@
+import test from 'ava';
+import sinon from 'sinon';
+
+import AudioBandwidthTest from '../../src/diagnostics/AudioBandwidthTest';
+
+let audioBandwidthTest, options;
+test.before(() => {
```

Comment:
test.before(t => {
	// This runs before all tests
});

test.before(t => {
	// This runs after the above, but before tests
});

test.after('cleanup', t => {
	// This runs after all tests
});

test.after.always('guaranteed cleanup', t => {
	// This will always run, regardless of earlier failures
});

test.beforeEach(t => {
	// This runs before each test
});

test.afterEach(t => {
	// This runs after each test
});

test.afterEach.always(t => {
	// This runs after each test and other test hooks, even if they failed
});

test(t => {
	// Regular test
});

So before runs before all the tests, so you propose doing test.beforeEach then